.travis.yml

```diff
@@ -25,6 +25,7 @@ before_deploy:
 - >
   if ! [ "$BEFORE_DEPLOY_RUN" ]; then
     export BEFORE_DEPLOY_RUN=1;
+    export ASCIIDOCTOR_CORE_VERSION=v1.5.6;
```

Comment:
`before_deploy` is triggered for each `deploy` providers, this is why I created `BEFORE_DEPLOY_RUN`.

It's done after all the builds (matrix) but I'm not sure.

In this context the initial `ASCIIDOCTOR_CORE_VERSION` defined by the matrix is not really relevant.